src/components/common/Alpha.jsx

```diff
@@ -76,17 +72,6 @@ class Alpha extends ReactCSS.Component {
     }
   }
 
-  handleMouseDown(e) {
-    this.handleChange(e, true);
-    window.addEventListener('mousemove', this.handleChange);
```

Comment:
You could probably use a combo of `onMouseDown  => this.state.isSelecting = true` `[onMouseDown, onMouseMove] => this.state.isSelecting ? onChange` `[onMouseUp, onMouseOut] => this.state.isSelecting = true`.

But idk if that's actually better.